public/js/index.js

```diff
@@ -3096,11 +3096,10 @@ function deleteDependencyModal(id, type) {
 }
 
 function initIssueList() {
-    const repolink = $('#repolink').val();
     $('#new-dependency-drop-list')
         .dropdown({
             apiSettings: {
-                url: suburl + '/api/v1/repos/' + repolink + '/issues?q={query}',
+                url: suburl + '/api/v1/repos/issues/search?q={query}',
```

Comment:
I think you should pass the repo id or name, and prioritize issues from that repository. There can be thousands of issues in a Gitea installation (Gitea alone has passed 8200).

Reply:
thanks, this is done now